ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -104,8 +157,18 @@ function get_hawq_password() {
   done
 }
 
+function sync_rps_configuration() {
+  if [[ -d $GPHOME/ranger/etc/ && ! -z $HAWQ_STANDBY_HOST ]]; then
+    cmd="scp $GPHOME/ranger/etc/* $HAWQ_STANDBY_HOST:$GPHOME/ranger/etc/"
```

Comment:
Reasonable.
But there are some other stuffs(rps ha tests,docs) assumed the script should copy rps configuration files to standby...
I think copy is also ok, and we can finish other stuffs quickly.

In future, we will go back to consider whether remove copy is more easy-to-use for user.